pandas/core/ops/array_ops.py

```diff
@@ -427,15 +427,15 @@ def maybe_upcast_datetimelike_array(obj: ArrayLike) -> ArrayLike:
     return obj
 
 
-def _maybe_upcast_for_op(obj, shape: Tuple[int, ...]):
+def _maybe_upcast_for_op(obj, shape: Shape):
     """
     Cast non-pandas objects to pandas types to unify behavior of arithmetic
     and comparison operations.
 
     Parameters
     ----------
     obj: object
-    shape : tuple[int]
+    shape : tuple[int] or int
```

Comment:
looks like the annotation and docstring dont match?

Reply:
Right. Reverting the change to the docstring